src/frontend/components/UI/CachedImage/index.tsx

```diff
@@ -8,7 +8,10 @@ interface CachedImageProps {
 type Props = React.ImgHTMLAttributes<HTMLImageElement> & CachedImageProps
 
 const CachedImage = (props: Props) => {
-  const [useCache, setUseCache] = useState(props.src.startsWith('http'))
+  const srcLocation = props.src ?? props.fallback
```

Comment:
This also doesn't look right, `props.src` is of type `string`, so (assuming everything's following the type definitions), the `?? props.fallback` should never happen

Reply:
Yes, in some cases the props.src is coming as undefined, I believe this happens when some game does not have a keyImage key on the metadata. 
I think this check here causes no harm and is better to keep it unless we are sure that all keyImages will come correctly.

Reply:
> Yes, in some cases the props.src is coming as undefined, I believe this happens when some game does not have a keyImage key on the metadata.
I think this check here causes no harm and is better to keep it unless we are sure that all keyImages will come correctly.

Right, but the only reason TS is not yelling at you there is because it thinks `props.src` will always be a `string` (try changing the type of it to `string | undefined`, you'll see what I mean)
Either `props.src` or `props.fallback` should be *just* `string` for this to be safe. If `props.src` isn't, `props.fallback` should be